plugin/neogit.lua

```diff
@@ -21,10 +21,6 @@ end, {
   desc = "Prints neogit message history",
 })
 
-api.nvim_create_user_command(
-  "NeogitResetState",
-  function()
-    require("neogit.lib.state")._reset()
-  end,
-  { nargs = "*", desc = "Reset any saved flags" }
-)
+api.nvim_create_user_command("NeogitResetState", function()
+  require("neogit.lib.state")._reset()
+end, { nargs = "*", desc = "Reset any saved flags" })
```

Comment:
This was formatted by Stylua :shrug:

Reply:
Sometimes when it does a crap job, I'll stick an ignore comment above and just do myself 🤓

Reply:
So long as my formatter and linter are happy then I'm happy. I haven't run into Stylua making lua code too mangled, so I tend to just run with what it outputs. Let me know if you want me to stick an ignore line there and put that block back to the original.

Reply:
It's pretty alright, but lacks nuance that other linters have. I'd kill for allowing lining up equal signs.. alas.

Reply:
I hear that if you light candles in a circle around 3am and chant "Mike Pall, Mike Pall, align my code, I beg of you!" five times fast Mike Pall descends from Lua heaven to align the electrons by hand on your drive while screaming "Stylua!" to get your whitepsace looking pretty.

Reply:
Thats a tier in leet-code I haven't gotten to yet XD